ansible_ai_connect/ai/api/tests/test_completion_view.py

```diff
@@ -534,49 +432,8 @@ def test_full_payload_without_ansible_lint_with_commercial_user(self):
                 self.assertSegmentTimestamp(log)
 
     @override_settings(ENABLE_ANSIBLE_LINT_POSTPROCESS=True)
-    @override_settings(ENABLE_ARI_POSTPROCESS=False)
-    @override_settings(SEGMENT_WRITE_KEY="DUMMY_KEY_VALUE")
-    def test_full_payload_with_ansible_lint_without_ari_postprocess_with_commercial_user(self):
```

Comment:
This test should remain, you can rename it and drop the `without_ari_` from the test name